source/prerequisites/sybase.txt

```diff
@@ -0,0 +1,15 @@
+.. _rm-prereq-mysql:
```

Comment:
[Question/Suggestion]

If there are no prereqs for Sybase, do we need this page? Not sure if this is in scope or if you were already on planning to do this but maybe we could address the fact that there is no Sybase requirements on the prereq landing page?

Reply:
People won't know there are no prerequisites. Although we should use this page to explain that Sybase only supports snapshots, not continuous jobs.